app/assets/src/components/ProjectSelection.jsx

```diff
@@ -10,6 +11,10 @@ import Samples from './Samples';
  class ProjectSelection extends React.Component {
   constructor(props) {
     super(props);
+    this.nanobar = new Nanobar({
```

Comment:
wouldn't this collide with the Samples nano-bars with the same id?

Reply:
yea, the nano-bar takes the same position at the top of the page, so the id should be the same. @yunfangjuan.  It is basically the same nano-bar.